{
  "tweet": {
    "edit_info": {
      "initial": {
        "editTweetIds": [
          "564084197326061569"
        ],
        "editableUntil": "2015-02-07T16:32:07.477Z",
        "editsRemaining": "5",
        "isEditEligible": true
      }
    },
    "retweeted": false,
    "source": "<a href=\"http://twitter.com\" rel=\"nofollow\">Twitter Web Client</a>",
    "entities": {
      "hashtags": [
        {
          "text": "python",
          "indices": [
            "45",
            "52"
          ]
        }
      ],
      "symbols": [],
      "user_mentions": [],
      "urls": []
    },
    "display_text_range": [
      "0",
      "126"
    ],
    "favorite_count": "0",
    "id_str": "564084197326061569",
    "truncated": false,
    "retweet_count": "1",
    "id": "564084197326061569",
    "created_at": "Sat Feb 07 15:32:07 +0000 2015",
    "favorited": false,
    "full_text": "Uh, why does reading one byte from a file in #python trigger a UnicodeDecodeError considering I never said to decode anything?",
    "lang": "en"
  }
}